src/pint/models/astrometry.py

```diff
@@ -550,8 +550,12 @@ def as_ECL(self, epoch=None, ecl="IERS2010"):
             ra=self.RAJ.quantity,
             dec=self.DECJ.quantity,
             obstime=self.POSEPOCH.quantity,
-            pm_ra_cosdec=self.RAJ.uncertainty * np.cos(self.DECJ.quantity) / dt,
-            pm_dec=self.DECJ.uncertainty / dt,
+            pm_ra_cosdec=self.RAJ.uncertainty * np.cos(self.DECJ.quantity) / dt
+            if self.RAJ.uncertainty is not None
+            else 0 * self.RAJ.units / dt,
+            pm_dec=self.DECJ.uncertainty / dt
+            if self.DECJ.uncertainty is not None
+            else 0 * self.DECJ.units / dt,
```

Comment:
Please put the if expressions inside parentheses for readability. I am surprised that the inline if expressions are formatted this way by black.